nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
```

Comment:
I don't understand the reasoning behind t_slice_begin and t_slice_end. There are both used for print_progress_() and not changed anywhere else. But what does timeval() return? Does this refer to "struct timeval" from the C standard library?